File: python/prime_numbers.py
n = int(input("enter a number: "))
list = []
for i in range (1 , n + 1) :
    if n % i == 0:
        list . append(i)
    else :
        continue
l = len(list)
if l == 2:
    print("the number is a prime number")
else :
    print("the number is not a prime number")

File: CP in c++/c/prime_numbers.c
#include<stdio.h>
int main(){
    int n;
    printf("enter a number: ");
    scanf("%d" , &n);
    int z;
    if (n == 1)
    {
        printf("neither prime nor composite");
    }
    else 
    {
        for (int i = n - 1 ; i >= 1 ; i--){
            if (n % i == 0){
                z = i;
                break;
            }
            else continue;
        }
        if (z == 1) printf("this is a prime number");
        else printf("this is a composite number");
    }
    return 0;
}